tests/test_mwss_transform.py

```diff
@@ -716,27 +716,16 @@ def test_pck_no_defaults(self):
         ], rv)
 
 
-# class PackingTests(unittest.TestCase):
-
-#     def test_requires_batch_nr(self):
-#         self.assertRaises(
-#             TypeError,
-#             MWSSTransformer,
-#             {},
-#             seq_nr=0
-#         )
-#         rv = CSFormatter.pck_lines(data, **ids._asdict())
-#         self.assertEqual([
-#             "FV          ",
-#             "0005:49900001225C:200911",
-#             "0040 00000000002",
-#             "0140 00000000124",
-#             "0151 00000217222",
-#         ], rv)
-
-
 class PackingTests(unittest.TestCase):
 
+    def test_requires_seq_nr(self):
```

Comment:
This test should be called `test_requires_ids`. The UserWarning is raised in https://github.com/ONSdigital/sdx-common/blob/master/sdx/common/transformer.py#L101